porcupine/settings.py

```diff
@@ -414,6 +414,7 @@ def _init_global_gui_settings() -> None:
     global_settings.add_option(
         "default_line_ending", LineEnding(os.linesep), converter=LineEnding.__getitem__
     )
+    global_settings.add_option("remember_tabs_on_restart", default=True)
```

Comment:
Defining this setting should move into the `restart` plugin

Reply:
@rdbende When you review this PR, feel free to fix this if I haven't fixed it yet :)